tox.ini

```diff
@@ -2,8 +2,8 @@
 minversion = 1.9
 envlist =
     lint
-    py{35,36,37,38}-django111-alchemy-mongoengine,
-    py{35,36,37,38}-django22-alchemy-mongoengine,
+    py{36,37,38}-django111-alchemy-mongoengine,
```

Comment:
I think we should also drop Django 1.11 :)

Reply:
Yep, done in #793.